API_request_test.py

```diff
@@ -3,6 +3,40 @@
 
 import requests
 import json
+import os, sys
+import argparse
+
+parser = argparse.ArgumentParser(
+                    prog='BetterWeather',
+                    description='This program is written in Python. Its purpose is to collect weather forecast data\n'
+                    'from a few websites (e.g. Open-Meteo, WeatherAPI) and combine it\n'
+                    'into one more accurate weather prediction. After providing the location eg. "Madrid"\n'
+                    'you will receive a file with weather data and a user-friendly plot.')
+
+# positional arguments
+parser.add_argument('location', metavar='location', type= str,
+                    help='enter location for which you want to get weather forecast e.g. Cracovia')
```

Comment:
```suggestion
                    help='Enter the location for which you want to get the weather forecast e.g. "Cracovia".')
```
Please always treat the help information as full sentences.